src/backbone.marionette.js

```diff
@@ -39,56 +37,52 @@ import {
 } from './config/features';
 
 const previousMarionette = Backbone.Marionette;
-const Marionette = Backbone.Marionette = {};
 
 // This allows you to run multiple instances of Marionette on the same
 // webapp. After loading the new version, call `noConflict()` to
 // get a reference to it. At the same time the old version will be
 // returned to Backbone.Marionette.
-Marionette.noConflict = function() {
+export const noConflict = function() {
   Backbone.Marionette = previousMarionette;
   return this;
 };
 
 // Utilities
-Marionette.bindEvents = proxy(bindEvents);
-Marionette.unbindEvents = proxy(unbindEvents);
-Marionette.bindRequests = proxy(bindRequests);
-Marionette.unbindRequests = proxy(unbindRequests);
-Marionette.mergeOptions = proxy(mergeOptions);
-Marionette.getOption = proxy(getOption);
-Marionette.normalizeMethods = proxy(normalizeMethods);
-Marionette.extend = extend;
-Marionette.deprecate = deprecate;
-Marionette.triggerMethod = proxy(triggerMethod);
-Marionette.isEnabled = isEnabled;
-Marionette.setEnabled = setEnabled;
-Marionette.monitorViewEvents = monitorViewEvents;
-Marionette.BackboneViewMixin = BackboneViewMixin;
-
-// Classes
-Marionette.Application = Application;
-Marionette.AppRouter = AppRouter;
-Marionette.View = View;
-Marionette.CollectionView = CollectionView;
-Marionette.Behavior = Behavior;
-Marionette.Region = Region;
-Marionette.Error = MarionetteError;
-Marionette.Object = MarionetteObject;
+export const bindEvents = proxy(_bindEvents);
+export const unbindEvents = proxy(_unbindEvents);
+export const bindRequests = proxy(_bindRequests);
+export const unbindRequests = proxy(_unbindRequests);
+export const mergeOptions = proxy(_mergeOptions);
+export const getOption = proxy(_getOption);
+export const normalizeMethods = proxy(_normalizeMethods);
+export const triggerMethod = proxy(_triggerMethod);
 
 // Configuration
-Marionette.DEV_MODE = false;
-Marionette.FEATURES = FEATURES;
-Marionette.VERSION = version;
-Marionette.DomApi = DomApi;
-Marionette.setDomApi = function(mixin) {
+
+export const setDomApi = function(mixin) {
   CollectionView.setDomApi(mixin);
   Region.setDomApi(mixin);
   View.setDomApi(mixin);
 };
-Marionette.setRenderer = function(renderer) {
+export const setRenderer = function(renderer) {
   CollectionView.setRenderer(renderer);
   View.setRenderer(renderer);
 };
 
-export default Marionette;
+export {
+  View,
+  CollectionView,
+  MnObject,
+  Region,
+  Behavior,
+  Application,
+  AppRouter,
+  isEnabled,
+  setEnabled,
+  monitorViewEvents,
+  BackboneViewMixin,
+  extend,
+  DomApi,
+  version as VERSION,
+  FEATURES
```

Comment:
I think we can drop this export.  It isn't documented and there's a clear API for how to enable it.  I assume this is here because originally things were attached to the global `Marionette` a lot pre ES6 modules